lua/telescope/_extensions/session-lens/session-lens-library.lua

```diff
@@ -93,7 +93,19 @@ do
     mt_file_entry.display = function(entry)
       -- Revert session files back to paths for easier reading
       -- This affects the display only, a selection still uses the correct file name.
-      local display = path.make_relative((entry.value:gsub("%%", "/")), cwd)
+      local display = ''
+      if vim.fn.has('win32')==1 then
+        local entry_value = entry.value:gsub("%++", ":")
+        entry_value = entry_value:gsub("%%", "/")
+        display = path.make_relative((entry_value), cwd)
```

Comment:
I'd just break this out into it's own `get_win32_display` function that returns the display value as well as extract the `vim.fn.has('win32')` into it's own constant and just use it like this
```lua
local WIN32 = vim.fn.has('win32')
display = WIN32 and get_win32_display() or path.make_relative((entry.value:gsub("%%", "/")), cwd)
```

Reply:
I agree with your idea. My first idea is to make the code easy to read and simple, because the code is not very long.